src/Umbraco.Core/Models/ContentTypeBase.cs

```diff
@@ -494,9 +494,9 @@ protected override void PerformDeepClone(object clone)
             }
         }
 
-        public IContentType DeepCloneWithResetIdentities(string alias)
+        public ContentTypeBase DeepCloneWithResetIdentities(string alias)
```

Comment:
Should this be `IContentTypeBase` ? or maybe doesn't matter

Reply:
Not sure it matters a whole bunch. I can change it if you feel strongly for it 😄